clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -75,6 +82,8 @@
 
     /* max time in ms for the producer to wait for acknowledgement from server*/
     private final int requestTimeoutMs;
+    
+    private final Time time;
```

Comment:
@hachikuji It is a good suggestion. However, I think it's best if I open a new Jira to address the issue of an update-able "Now" parameter throughout the code and limit the changes in this pull request.